kvui.py

```diff
@@ -386,6 +386,60 @@ def insert_text(self, substring, from_undo=False):
         return super(ConnectBarTextInput, self).insert_text(s, from_undo=from_undo)
 
 
+def is_command_input(string: str) -> bool:
+    return string and string[0] in "/!"
+
+
+class CommandPromptTextInput(TextInput):
+    MAXIMUM_HISTORY_MESSAGES = 50
+
+    def __init__(self, *args, **kwargs) -> None:
+        self._command_history_index = -1
+        self._command_history: typing.List[str] = []
+        return super().__init__(*args, **kwargs)
```

Comment:
Is there any reason to not call the parent constructor first (some other languages actually require you to do that as the first action)?

Reply:
It shouldn't make a difference here, but I can make that adjustment.